R/constants.R

```diff
@@ -392,12 +394,13 @@ emissions.GAINS_BASE_YEAR <- 2005
 emissions.GAINS_YEARS <- c(2010, 2020, 2030)
 emissions.LOW_PCGDP <- 2.75
 emissions.COAL_SO2_THRESHOLD <- 0.1
-
-emissions.EPA_MACC_YEAR        <- 2030  # Must be either 2020 or 2030
```

Comment:
?

Reply:
This was a duplicate line.  The constant `emissions.EPA_MACC_YEAR` is listed on line 386.
Should I revert?